views/profile.handlebars

```diff
@@ -10,6 +10,15 @@
       <p>(User's posted items here maybe?)</p>
       <p> item item item item item item item item item item item item</p>
 
+      {{!-- Button to create a new product from user's profile page, for testing purposes --}}
+      <form class="form create-product">
+        <div class="form-group">
+          <label for="product-info">Post a new product</label>
+          <input class="form-input" type="text" id="product-info" />
+          <button class="btn btn-primary" type="submit">Create</button>
+        </div>
+      </form>
```

Comment:
Users can post a new product from their profile page using a "create" button. Currently doesn't take in any real input from the input field and just creates a dummy product.